examples/tp/hello_tp.py

```diff
@@ -39,13 +39,19 @@
 # Can't live without numpy
 import numpy
 
+# izip for maximum efficiency
+from itertools import izip as zip, count
+
 # here we choose TP implementation: (uncomment one only)
-# This is the class corresponding to the C++ optimized Temporal Pooler (default)
-from nupic.research.TP10X2 import TP10X2 as TP
+# This is the class corresponding to the C++ optimized Temporal Pooler(default)
+# from nupic.research.TP10X2 import TP10X2 as TP
```

Comment:
does this code still run with TP? it would be nice if it did, if not, you can remove the comments here.